### PR TITLE
[ADD][11.0] account_invoice_reimbursable

### PR DESCRIPTION
**Pull request by etobella**
Reimbursables management
========================

This module adds the option to create reimbursables.

Reimbursables are charges made to the customer on behalf of a third party.

The customer will receive two invoices, one from the supplier that includes
the reimbursable and one from the third party.

The supplier will pay to the third party its invoice to the customer on behalf
of the customer

Usage
=====

#. Go to 'Accounting/Invoicing > Purchases > Documents > Vendors Bills'
#. Create an invoice for a provider and add the reimbursables on the
   reimbursable page. Reimbursable partner will be required
#. Validate the invoice

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Is this the Spanish "suplido"?

**Comment by etobella** [comment]
I am still working on it, because, it only works for supplier invoices. It should be fixed in order to be used on customer invoices and use the elements for the payments.

**Comment by etobella** [comment]
Yes, it is the spanish "suplido"

**Comment by pedrobaeza** [comment]
Great! I will say it for having reviewers in this.

**Comment by etobella** [comment]
I think that is outside the scope of this PR the usage of customer invoices for some reasons:
-  A lot of companies receive invoices with reimbursements (usually when lawyers are involved), but not many make invoices with reimbursements
-  Reimbursements on customer invoices are hard to deal for reconciliation. I think that it needs to be meditated

So, I think the best option is to make hook in order to allow the change of accounts on the second module. What do you think?

Also, I had to add a modification of an account method until odoo/odoo#24915 is applied

**Comment by etobella** [comment]
I finally removed all the unnecessary code and I think it is better to maintain customer invoices out of the current goal of this module.
It is ready for testing now!

**Review by JordiBForgeFlow — APPROVED** [review]
:+1:  Technical and functional review.

File: oca_dependencies.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (MiquelRForgeFlow) [inline oca_dependencies.txt:1]: why?

Comment (etobella) [reply oca_dependencies.txt:1]: Travis was returning a small issue, not an error, but I tried to solve it.

**Review by MiquelRForgeFlow — APPROVED** [review]
Checked on runbot

**Pull request merged**